aiocogeo/cog.py

```diff
@@ -117,13 +117,17 @@ def profile(self) -> Dict[str, Any]:
             "photometric": self.photometric,
         }
 
+    @property
+    def indexes(self) -> Tuple[int]:
+        return (r + 1 for r in range(self.ifds[0].bands))
+
     @property
     def epsg(self) -> int:
         """Return the EPSG code representing the crs of the image"""
         return self.ifds[0].geo_keys.epsg
 
     @property
-    def bounds(self) -> Tuple[float, float, float, float]:
+    def geo_bounds(self) -> Tuple[float, float, float, float]:
```

Comment:
same as for point and preview, bounds is already a property in AsyncBaseReader.

Note: could use a better name